packages/web-components/fast-components-msft/src/anchor/index.ts

```diff
@@ -21,4 +28,23 @@ import { AnchorStyles as styles } from "./anchor.styles";
         delegatesFocus: true,
     },
 })
-export class FASTAnchor extends Anchor {}
+export class FASTAnchor extends Anchor {
+    /**
+     * The appearance the anchor should have.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: appearance
+     */
+    @attr
```

Comment:
Have considered doing this without actually adding the attribute? This could be done with a host+attribute selector and would be "cheaper" at runtime.

Reply:
This is somewhat step 1 of 2 - the next step is going to be moving away from the styles being associated to a class and attaching the appearance stylesheet based on the appearance "behavior". I could be wrong, but it seems like that would throw a wrinkle in the above.